test/prism/ripper_test.rb

```diff
@@ -255,7 +255,6 @@ def test_alias
       assert_equivalent("alias :'' :foo")
     end
 
-    Translation::Ripper
```

Comment:
This is here to autoload `RipperCompiler`.

Reply:
It causes this warning when running tests:
```
$ be rake
make
make: Nothing to be done for 'all'.
/usr/bin/gmake install sitearchdir=../../../../lib/prism sitelibdir=../../../../lib/prism target_prefix=
/usr/bin/install -c -m 0755 prism.so ../../../../lib/prism
cp tmp/x86_64-linux/prism/3.3.0/prism.so tmp/x86_64-linux/stage/lib/prism/prism.so
/home/eregon/code/yarp/test/prism/ripper_test.rb:258: warning: possibly useless use of :: in void context
Loaded suite /home/eregon/code/yarp/vendor/bundle/ruby/3.3.0/gems/rake-13.1.0/lib/rake/rake_test_loader
Started
Finished in 5.248438505 seconds.
------------------------------------------------------------------------------------------------------------------------------
2576 tests, 529234 assertions, 0 failures, 0 errors, 0 pendings, 0 omissions, 0 notifications
100% passed
------------------------------------------------------------------------------------------------------------------------------
490.81 tests/s, 100836.47 assertions/s
```
I'll just remove this commit for now.
Maybe `RUBY_KEYWORDS` could be frozen and assigned on `Translation::Ripper` as well to fix it?
Or use it later, only after a new `assert_equivalent`  (which loads `Translation::Ripper`) in test_keyword_aliases.